src/models/item.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const itemSchema = new Schema({
+  name: {
+    type: String,
+    maxLength: [30, "item name's max length is 30 characters"],
+    required: [true, 'Item name is required'],
+  },
+  type: {
+    type: String,
+    enum: {
+      values: [
+        'Books',
+        'Stationary',
```

Comment:
Please change "Stationary" to "Stationery".

Note: Stationary means “fixed,” “immobile,” or “unchanging.” Stationery refers to paper, matching envelopes, and writing implements.

Reply:
Done, thanks for the note 😁